.changeset/kind-camels-give.md

```diff
@@ -0,0 +1,12 @@
+---
+"@codecov/bundler-plugin-core": patch
+"@codecov/nuxt-plugin": patch
+"@codecov/remix-vite-plugin": patch
+"@codecov/rollup-plugin": patch
+"@codecov/solidstart-plugin": patch
+"@codecov/sveltekit-plugin": patch
+"@codecov/vite-plugin": patch
+"@codecov/webpack-plugin": patch
+---
+
+Remove requirement for uploadToken being present adding support for tokenless uploads.
```

Comment:
h: We need to update the README of the packages as well

Reply:
Hrrm trying to think of a nice way to word "this", do you think having two examples or just adjusting the current codeblock to have the same options but just commenting around the differences would be better?

Reply:
I think we should make tokenless the default snippet, and then add an extra section below about what to do if you need to use the token, but keep that at the bottom.